backend/models/applicant/model.py

```diff
@@ -0,0 +1,36 @@
+from odmantic import Field, Model
+from datetime import date
+from typing import Optional, List, IO
+
+
+class WorkExperience(Model):
+    companyName: str
+    dateFrom: date
+    dateTo: date
+    description: str
+
+class Education(Model):
```

Comment:
Can you change `(Model)` to `(EmbeddedModel)` instead? This will treat the `Education` and `WorkExperience` models as submodels for the `Applicant`'s .

Reply:
I've adjusted the code, though, for the Job model, it has a field that relates to the organisation model, should I change the organisation model to an embedded model or is it alright if we can leave it as a normal model